### PR TITLE
Add Asset Tracking JavaScript SDK docs

### PR DESCRIPTION
**Pull request by marklewin**
> _NOTE TO REVIEWERS_ - please **do not review PRs in the `DRAFT` state**, as the PR may change substantially before it is ready to review. Thanks.

## Description

An initial attempt to document the Asset Tracking JavaScript SDK and associated example app.

* [DOC-379](https://ably.atlassian.net/browse/DOC-379).

## Review

Instructions on how to review the PR. 

* [Overview](https://ably.com/documentation/asset-tracking) - should show availability of Publishing and Subscribing SDKs by platform
* [Using the example apps](https://ably.com/documentation/asset-tracking/example-apps) - check JavaScript SDK section and verify that you can install and run using the provided instructions
* [Using the SDKs](https://ably.com/documentation/asset-tracking/using-the-sdks) - check that the JavaScript and Android sections are formatted correctly when selecting the page language, and that the JavaScript sections are clear and technically correct.

### CODE REVIEW

File: content/asset-tracking/using-the-sdks.textile [hunk 1]
```diff
@@ -115,96 +126,105 @@ val publisher = Publisher.publishers() // get the Publisher builder in default s
...
+  clientId: 'CLIENT_ID'
+})
+```
 
-h2(#publishing-sdk). Using the publishing SDK
+You can read more about JWT authentication in our "documentation":/core-features/authentication#ably-jwt-process.
 
-Common operations you need to carry out on the publisher include:
+blang[android].
+  h2(#publishing-sdk). Using the publishing SDK
```
Comment (m-hulbert) [inline content/asset-tracking/using-the-sdks.textile:140]: "Subscribing SDK" is capitalised so "publishing SDK" should be too.

**Review by m-hulbert — APPROVED** [review]
Reads well and tested in the web view. I've made some minor suggestions to conform to the style guidelines and for consistency, but approving for the content.

**Review by tbedford — APPROVED** [review]
I've only tested how things look under web view, and I'm happy with how we are handling multiple languages.

File: content/asset-tracking/using-the-sdks.textile [hunk 2]
```diff
@@ -74,24 +72,28 @@ The client requires authentication in order to establish a connection with Ably.
 
 Usually a client will use either token or JWT authentication, as Basic Authentication would require exposing the API keys on the client.
 
-Examples of establishing a connection using the three methods are given in the following sections. While the examples shown are for the publishing SDK, the code equally applies to establishing a connection using the Subscribing SDK.
+Examples of establishing a connection using the three methods are given in the following sections. While the examples shown are for either the Publishing or Suscribing SDK, you can use the same approach for both SDKs.
 
 h3(#basic-authentication). Basic Authentication
 
-The following example demonstrates establishing a connection using Basic Authentication:
+The following example demonstrates establishing a connection using basic authentication:
```
[the remark this answers is not in the document]
Comment (marklewin) [reply content/asset-tracking/using-the-sdks.textile:79]: Fixed

File: content/asset-tracking/using-the-sdks.textile (hunk 1 above)
Comment (marklewin) [reply content/asset-tracking/using-the-sdks.textile:140]: Yes it should! Fixed.

File: content/asset-tracking/using-the-sdks.textile [hunk 3]
```diff
@@ -239,6 +265,25 @@ val subscriber = Subscriber.subscribers() // Get an AssetSubscriber
     .start() // start listening for updates
 ```
 
+```[javascript]
+// Initialize the Subscriber
+import { Subscriber, Accuracy } from '@ably/asset-tracking';
+
+const ablyOptions = {
+  key: "ABLY_API_KEY",
+  clientId: "CLIENT_ID",
```
[the remark this answers is not in the document]
Comment (marklewin) [reply content/asset-tracking/using-the-sdks.textile:274]: Agree. Fixed.

File: content/asset-tracking/using-the-sdks.textile [hunk 4]
```diff
@@ -261,9 +317,24 @@ subscriber.trackableStates
     .launchIn(scope) // coroutines scope on which the statuses are received
 ```
 
+```[javascript]
+// Listen for asset state changes
+subscriber.onLocationUpdate((isOnline) => {
+  console.log(
+    `Status update: Publisher is now ${isOnline ? "online" : "offline"}`
```
[the remark this answers is not in the document]
Comment (marklewin) [reply content/asset-tracking/using-the-sdks.textile:324]: Done

File: content/asset-tracking/example-apps.textile [hunk 5]
```diff
@@ -5,64 +5,113 @@ index: 10
...
+  npm install
+  ```
 
-<a href="/images/screenshots/asset-tracking/android/select-app.png" target="_blank">
-  <img src="/images/screenshots/asset-tracking/android/select-app.png" style="width: 100%" alt="Select the publishing app">
-</a>
+  5. Create a @.env@ file in the root of the example app and configure it with your Ably API key:
 
-The app builds and the emulator is displayed. It takes a while to boot up.
+  ```
```
[the remark this answers is not in the document]
Comment (marklewin) [reply content/asset-tracking/example-apps.textile:92]: I don't see this behaviour in either the docs or the web view?

[the remark this answers is not in the document]
Comment (marklewin) [reply content/asset-tracking/example-apps.textile:92]: Ah, I see it now. Yes, let's stick a `<br>` in there for now.

**Pull request merged**